src/bsoncxx/oid.hpp

```diff
@@ -114,12 +117,9 @@ class BSONCXX_API oid {
     std::time_t get_time_t() const;
 
     ///
-    /// @todo document this method. Also, do we really want this? This is the only place we use ostreams.
-    ///
-    friend BSONCXX_API std::ostream& BSONCXX_CALL operator<<(std::ostream& out, const oid& rhs);
-
+    /// An accessor for the internal data buffer in the oid.
     ///
-    /// @todo document this method
+    /// @return A pointer to the internal buffer holding the oid bytes.
     ///
     const char* bytes() const;
```

Comment:
With std::array as the implementation, should this just return `const std::array<...>&